src/helm/benchmark/scenarios/numeracy_scenario.py

```diff
@@ -557,6 +559,82 @@ def get_numeracy_adapter_spec(
     )  # enable override
 
 
+class NumeracyAdapter:
```

Comment:
I think code structure wise, this should appear in its own file in the src/helm/benchmark/adaptation/adapters/ directory, and inherit from Adapter.

Also, I believe the style is to make Scenario and Adapter peers ([example](https://github.com/stanford-crfm/helm/blob/8ea285f7f294165fae0f737a082769e25099d333/src/helm/benchmark/run_specs.py#L800-L808)) such that the Scenario outputs Instance objects which the Adapter can take in to make ScenarioState.